src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Processors/DependencyFilteringTelemetryProcessor.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+
+namespace Microsoft.Azure.WebJobs.Logging.ApplicationInsights
+{
+    internal class DependencyFilteringTelemetryProcessor : ITelemetryProcessor
+    {
+        private readonly ITelemetryProcessor _next;
+
+        public DependencyFilteringTelemetryProcessor(ITelemetryProcessor next)
```

Comment:
`DependencyFilteringTelemetryProcessor` may be confusing as this processor filters *requests* and dependencies,   how about `OperationTelemteryFilteringProcessor`?

Reply:
Good point. Will update.